geocoding_platform_interface/lib/src/models/placemark.dart

```diff
@@ -97,12 +97,12 @@ class Placemark {
       thoroughfare.hashCode;
 
   /// Converts a list of [Map] instances to a list of [Placemark] instances.
-  static List<Placemark> fromMaps(dynamic message) {
+  static List<Placemark>? fromMaps(dynamic message) {
```

Comment:
Same as with `Location`, I would not expect a `null` value. Maybe an empty list if converting the `message` to a `Placemark` doesn't work.